client/signup/steps/page-picker/index.tsx

```diff
@@ -199,36 +213,85 @@ const StyledButton = styled( Button )`
 	}
 `;
 
-export default function DIFMPagePicker( props: StepProps ) {
-	const { stepName, goToNextStep } = props;
+const Placeholder = styled.div`
```

Comment:
This causes an error:
<img width="1401" alt="image" src="https://user-images.githubusercontent.com/5436027/180158982-deeb41e1-d89c-41e0-8d82-273dbe528b90.png">
```suggestion
const Placeholder = styled.span`
```